### PR TITLE
Tensorflow frontend API:  tensor flow subtract math function

### PR DESCRIPTION
**Pull request by bezalel001**
Added tensorflow `subtract` math function to tensorflow frontend
#2559

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py [hunk 1]
```diff
@@ -189,3 +189,36 @@ def test_tensorflow_full(
         value=fill_value,
         rtol=1e-05,
     )
+
+
+# subtract
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(set(ivy_tf.valid_float_dtypes))
```
Comment (jkeane508) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py:198]: We should use `ivy_np.valid_numeric_dtypes `and `ivy_tf.valid_numeric_dtypes`. As subtract will accept int dtypes

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Looks good just this small change needed :)

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py (hunk 1 above)
Comment (bezalel001) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py]: Hi @jkeane508
Fixed!
Please review

Comment (jkeane508) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py]: Hey, you've actually changed both to `ivy_np.valid_numeric_dtypes` when we should intersect both `ivy_np.valid_numeric_dtypes` and `ivy_tf.valid_numeric_dtypes`

Comment (bezalel001) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py]: Oops! my bad!

**Comment by jkeane508** [comment]
Yep looks good to me!

Thanks!

**Pull request merged**